quality_cuts/new_var_dist.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from tree_importer import tree_importer
+import gc
+
+"""
+ All the nonesences ( $\chi$2 < 0), inf and nan were deleted. Also applied
+ quality cuts based on detector geometry. Full description could be found in
+ https://docs.google.com/document/d/11f0ZKPW8ftTVhTxeWiog1g6qdsGgN1mlIE3vd5FHLbc/edit?usp=sharing
+
+"""
+
+from concurrent.futures import ThreadPoolExecutor
+executor = ThreadPoolExecutor(8)
```

Comment:
remove these if you don't use them, they are already included in the tree_importer file